src/modelfree/common/utils.py

```diff
@@ -346,10 +347,13 @@ def simulate(venv, policies, render=False):
         yield observations, rewards, dones, infos
 
 
-def make_env(env_name, seed, i, out_dir, our_idx=None, pre_wrapper=None, post_wrapper=None):
+def make_env(env_name, seed, i, out_dir, our_idx=None, pre_wrapper=None, post_wrapper=None,
```

Comment:
Change `pre_wrapper` and `post_wrapper` to lists: `pre_wrappers` and `post_wrappers`, iterate over the lists (if not `None`). Then you can just add `OldMujocoResettableWrapper` to the `pre_wrapper` list, no need to have `make_env` be aware of this.